src/Paket.Core/PackageMetaData.fs

```diff
@@ -167,17 +167,39 @@ let findDependencies (dependenciesFile : DependenciesFile) config platform (temp
                     
                 deps, p :: files) ([], [])
     
-    // Add the assembly + pdb + dll from this project
+    // Add the assembly + {.dll, .pdb, .xml, /*/.resources.dll} from this project
     let templateWithOutput =
+        let projects =                 
+            if includeReferencedProjects then 
+                project.GetAllInterProjectDependenciesWithoutProjectTemplates() 
+                |> Seq.toList 
+            else 
+                [ project ]
+
+        let satelliteDlls =
+            seq {
+            for project in projects do
+                let satelliteAssemblyName = Path.GetFileNameWithoutExtension(project.GetAssemblyName()) + ".resources.dll"
+                let projectDir = Path.GetDirectoryName(Path.GetFullPath(project.FileName))
+                let outputDir = Path.Combine(projectDir, project.GetOutputDirectory config platform)
+                for language in project.FindLocalizedLanguageNames() do
+                    let fileName = Path.Combine(outputDir, language, satelliteAssemblyName)
+                    if File.Exists fileName then
+                        let satelliteTargetDir = Path.Combine(targetDir, language)
+                        yield (FileInfo(fileName), satelliteTargetDir)
```

Comment:
I could probably remove the `FileInfo` as it is just an allocation as we have already checked `Exists`